### PR TITLE
🤖 Add file patterns to config.json

### PR DESCRIPTION
**Pull request by ErikSchierboom**
**⚠ This PR requires you to make a simple change before merging. ⚠**

---

To save maintainers from having to manually specify the `files` key in their exercises' `.meta/config.json` files, we are providing support for track-level patterns. See [this PR](https://github.com/exercism/docs/pull/58) for details.

This PR adds (**purposefully wrong**) file patterns to the `config.json` file. It is up to you, the track maintainers, to change these patterns to their correct value.

You can use the following placeholders:
- `%{kebab_slug}`: the `kebab-case` exercise slug (e.g. `bit-manipulation`)
- `%{snake_slug}`: the `snake_case` exercise slug (e.g. `bit_manipulation`)
- `%{camel_slug}`: the `camelCase` exercise slug (e.g. `bitManipulation`)
- `%{pascal_slug}`: the `PascalCase` exercise slug (e.g. `BitManipulation`)

We will soon update `configlet` to enable it to automatically populate the `.meta/config.json` file's `files` property, at which point we will then batch-PR updates to all tracks that have merged this PR.

## Tracking
https://github.com/exercism/v3-launch/issues/19

### CODE REVIEW

**Comment by wolf99** [comment]
Have an open question on slack about my changes... not quite sure if the makefile needs to be duplicated between stub and the solution. Also if the example files need to stay `example` or not (https://exercism-team.slack.com/archives/CR91YFNG3/p1614027093008400?thread_ts=1613575298.444300&cid=CR91YFNG3)

**Comment by ErikSchierboom** [comment]
@wolf99 I've responded on Slack

File: config.json [hunk 1]
```diff
@@ -978,13 +995,13 @@
...
     "execution_mode/compiled",
     "paradigm/procedural",
+    "typing/static",
     "platform/android",
     "platform/ios",
     "platform/linux",
     "platform/mac",
     "platform/windows",
     "runtime/language_specific",
-    "typing/static",
```
Comment (ryanplusplus) [inline config.json:987]: Why was `typing/static` reordered here? It looks like it's otherwise in alphabetical order.

Comment (wolf99) [reply config.json:987]: On the v3 website (currently) the first three tags are listed beneath the track name in the listing of tracks.
This means that "platform/android" is one of the listed items. This changes that, making the tag used more relevant to the language.

I based this change in the thought that although C can indeed be used on Android, Android is not necessarily required to create C programs.

Comment (ryanplusplus) [reply config.json:987]: :+1:

**Review by ryanplusplus — APPROVED** [review]

**Pull request merged**